views/css/index.css

```diff
@@ -181,4 +181,23 @@ body {
     align-items: center;
 }
 
+#signup-form {
```

Comment:
Hi Dave, could you please split the signup-specific CSS to a 'signup.css' file (as per Sams advice) - then just link that styling in the main index.html